src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -645,7 +644,7 @@ uint8_t P036_data_struct::display_scroll(ePageScrollSpeed lscrollspeed, int lTas
         ScrollingLines.Line[j].LastWidth = PixLengthLineOut; // while page scrolling this line is right aligned
       }
 
-      if ((PixLengthLineIn > getDisplaySizeSettings(disp_resolution).Width) && (fScrollTime > 0.0f))
+      if ((PixLengthLineIn > getDisplaySizeSettings(disp_resolution).Width) && (iScrollTime >= 0))
```

Comment:
should be `iScrollTime > 0` since you are dividing by `iScrollTime` later.

Reply:
Thanks, I was a bit off the mark while changing the code.